lib/Manager.php

```diff
@@ -389,6 +389,25 @@ public function getLeftOneToOneRoomsForUser(string $userId): array {
 		return $rooms;
 	}
 
+	public function removeUserFromAllRooms(IUser $user) {
```

Comment:
From my POV this should go into the ParticipantService. The manager is "legacy" and should be replaced by a proper RoomService and ParticipantService in the future, so new stuff should go into those services directly

Reply:
Remove UserFromAllRooms depend of Manager and when I put the Manager class on constructor of ParticipantService did recursive loop on dependency injection because the Manager require the ParticipantService class. To solve will be necessary move the dependency methods from Manager to other classes. This will be a big change.

Reply:
yeah got it already. So ignore for now